### DISCUSSION TITLE
安装依赖时，提示jpeg-js错误

### DISCUSSION THREAD

**Discussion by YipSzeho** [Tsuk1ko/CQ-picfinder-robot#351] [问题求助] [answered]
nodejs 版本v14.20.0
debian 11

# npm audit report

jpeg-js  <0.4.4
Severity: high
Infinite loop in jpeg-js - https://github.com/advisories/GHSA-xvf7-4v9q-58w6
fix available via `npm audit fix`
node_modules/jpeg-js
  @jimp/jpeg  <=0.12.0 || >=0.16.1
  Depends on vulnerable versions of jpeg-js
  node_modules/@jimp/jpeg
    @jimp/types  <=0.11.1-canary.891.908.0 || >=0.16.1
    Depends on vulnerable versions of @jimp/jpeg
    node_modules/@jimp/types

3 high severity vulnerabilities

**Comment by YipSzeho** [comment]
但好像不影响使用0v0

**Comment by Tsuk1ko** [comment, marked as answer]
不用管，不是错误
